web/src/app/modules/actions/modules/browser-version-checker/components/browser-version-checker.component.ts

```diff
@@ -4,39 +4,39 @@ import { Config } from '../../../../../config/config';
 import { detect } from 'detect-browser';
 
 @Component({
-  moduleId: module.id.toString(),
-  selector: 'browser-version-checker',
-  templateUrl: 'browser-version-checker.component.html',
-  styleUrls: ['browser-version-checker.component.css']
+    moduleId: module.id.toString(),
+    selector: 'browser-version-checker',
+    templateUrl: 'browser-version-checker.component.html',
+    styleUrls: ['browser-version-checker.component.css']
 })
 
 export class BrowserVersionChecker {
 
-  constructor(private translate: TranslateService) {
-  }
-
-  public get browserOutdated(): boolean {
-    let currentBrowser = detect();
-    if (currentBrowser === null || currentBrowser === undefined || currentBrowser.name === null || currentBrowser.name === undefined) {
-      return true;
+    constructor(private translate: TranslateService) {
     }
 
-    let supportedBrowsers = Config.SUPPORTED_BROWSERS;
-    for (let i = 0; i < supportedBrowsers.length; i++) {
-      const element = supportedBrowsers[i];
-      if (currentBrowser.name === element.browser) {
-        if (parseInt(currentBrowser.version.split('.')[0]) >= element.minVersion) {
-          return false;
+    public get browserOutdated(): boolean {
+        let currentBrowser = detect();
+        if (currentBrowser === null || currentBrowser === undefined || currentBrowser.name === null || currentBrowser.name === undefined) {
+            return true;
+        }
+
+        let supportedBrowsers = Config.SUPPORTED_BROWSERS;
+        for (let i = 0; i < supportedBrowsers.length; i++) {
+            const element = supportedBrowsers[i];
+            if (currentBrowser.name === element.browser) {
+                if (parseInt(currentBrowser.version.split('.')[0]) >= element.minVersion) {
```

Comment:
I suggest to extract "parseInt(currentBrowser.version.split('.')[0]" as variable and give it a speaking name (e.g. "browserMajorVersionNumber")